### ISSUE TITLE
run js css before everything

### ISSUE THREAD

**Issue by ImmanuelSamuel** [jupyter/notebook#1102]
I am trying to run some scripts before anything loads.
Right now going through the nbextensions route, my script gets run only after the notebook loads.
Is there a way I can run a script/css before everything (even before the body dom element gets rendered).

**Comment by Carreau** [comment]
No, it's not possible, because we need the body to be loaded to actually fetch your script and execute it. 
The question is what are you trying to achieve ?

**Comment by ImmanuelSamuel** [comment]
I need to hide the body before it gets rendered. Right now when I run a script which hides the body through the extensions. This results in a brief period where I see the toolbar and stuff before the body gets hidden.

If I can get a css style in the head element then that would take care of it right? I have a script that later turns it back on.

**Comment by JamiesHQ** [comment]
@ImmanuelSamuel : We're doing a little housekeeping on our issue log and noticed this thread from 2016. Were you able to find a solution to your issue? Please let us know so we can close this one. thanks!
